Formula/git-sync.rb

```diff
@@ -0,0 +1,42 @@
+class GitSync < Formula
+  desc "Clones a git repository and keeps it synchronized with the upstream"
+  homepage "https://github.com/kubernetes/git-sync#readme"
+  url "https://github.com/kubernetes/git-sync/archive/refs/tags/v3.6.0.tar.gz"
+  sha256 "0b65c3f414c98b13d66e99c86680c67348be50d578f88a0888720888b3fcd700"
+  license "Apache-2.0"
+
+  head do
+    url "https://github.com/kubernetes/git-sync.git", branch: "master"
+    depends_on "pandoc" => :build
+  end
+
+  depends_on "go" => :build
+
+  depends_on "coreutils"
+
+  conflicts_with "git-extras", because: "both install `git-sync` binaries"
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GOFLAGS"] = "-mod=vendor"
+    inreplace "cmd/#{name}/main.go", "\"mv\", \"-T\"", "\"#{Formula["coreutils"].opt_bin}/gmv\", \"-T\"" if OS.mac?
```

Comment:
Can you ask upstream to use commands that also work with BSD move?

Reply:
Yes - I'll try to get this wrapped up today.

Reply:
Bug report sent upstream (https://github.com/kubernetes/git-sync/issues/600)